R/tidyeval.R

```diff
@@ -113,6 +113,11 @@ dt_squash_call <- function(x, env, data, j = TRUE) {
       list(x[[2]], x[[3]])
     }))
     args <- lapply(args, dt_squash, env = env, data = data, j = j)
+    odd_index <- as.logical(seq_along(args) %% 2)
+    args[odd_index] <- lapply(
+      args[odd_index],
+      function(.x) if (isTRUE(.x)) call2("rep", TRUE, quote(.N)) else .x
```

Comment:
I think this would be cleaner to apply to `x[[2]]` above — then you wouldn't need to compute the odds index.

Reply:
Yep, makes sense. All set.